xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1150 @@
+      for clarification of terminology.</para>
+    </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.prerequisites">
+      <title>Prerequisites</title>
+
+      <para>Before you start, you need to</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>have a repository on GitHub.</para>
+        </listitem>
+
+        <listitem>
+          <para>have a package on an OBS Instance.</para>
+        </listitem>
+      </itemizedlist>
+    </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.supported_scms">
+      <title>Supported SCMs</title>
+
+      <para>We support the GitHub.com and GitLab.com instances.</para>
+
+      <para>We also support Self-Hosted instances from GitHub and GitLab. As
+      long as the network connectivity works, OBS will be able to interact
+      with that SCM.</para>
+    </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication">
+      <title>Token Authentication</title>
+
+      <para>OBS and GitHub need to talk to each other. Tokens are the way to
+      make this happen.</para>
+
+      <sect3 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication.how_to_authenticate_obs_with_scm">
+        <title>How to Authenticate OBS with SCMs</title>
+
+        <para>You have to create a GitHub <emphasis role="bold">personal
+        access token</emphasis>. OBS is going to use it to talk to
+        GitHub on your behalf.</para>
+
+        <para>The personal access token needs, at least, the following scopes
+        assigned:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>GitHub: repo</para>
+          </listitem>
+
+          <listitem>
+            <para>GitLab: api</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>Check <link
+        xlink:href="https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token">GitHub’s</link>
+        and <link
+        xlink:href="https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#creating-a-personal-access-token">GitLab</link>’s
+        documentation to learn how.</para>
+      </sect3>
+
+      <sect3 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication.how_to_authenticate_scm_with_obs">
+        <title>How to Authenticate SCMs with OBS</title>
+
+         <para>You have to create an OBS <emphasis role="bold">workflow
+        token</emphasis>. Github is going to use it to trigger actions on
+        OBS on your behalf.</para>
+
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication.how_to_authenticate_scm_with_obs.create_token">
+          <title>Create Token</title>
+
+          <para>You can create the OBS token via WebUI in <emphasis>Profile
+          &gt; Manage Your Tokens.</emphasis></para>
+
+          <para>You can also use <emphasis>osc</emphasis> for this, like
```

Comment:
I would remove the last part of this sentence, so it would be:

`You can also use osc for this:`